libsolidity/ast/Types.cpp

```diff
@@ -383,6 +383,38 @@ vector<UsingForDirective const*> usingForDirectivesForType(Type const& _type, AS
 
 }
 
+set<FunctionDefinition const*, ASTNode::CompareByID> Type::operatorDefinitions(
+	Token _token,
+	ASTNode const& _scope,
```

Comment:
Shouldn't you just always be scanning the scope of ``typeDefinition``?

Reply:
Now that operators are global, probably. It used to be different when they could be defined in contracts and libraries.

I thought about simplifying that part, but in the end I decided to keep it as is for various reasons. It works, there was still a small chance that operators would not be global and, finally, I will change it soon anyway. After the PR that stores operator definitions in a `SourceUnit` annotation, this function will just be returning stuff from that annotation.